app/cosmos/databases/applications.js

```diff
@@ -0,0 +1,21 @@
+const cosmosClient = require('../client')
+const { cosmosConfig } = require('../../config')
+
+const applicationsDatabase = async () => {
+  try {
+    const { database } = await cosmosClient.databases.createIfNotExists({
+      id: cosmosConfig.applicationsDatabase,
+      throughput: 400
+    })
+    await database.containers.createIfNotExists({
+      id: cosmosConfig.applicationsContainer,
+      partitionKey: { paths: ['/id'] }
+    })
+
+    return database
+  } catch (err) {
+    console.log(err)
```

Comment:
If this throws an error the app just silently ignores it, is this safe?

Reply:
Oversight, have updated to throw error if database cannot be created

Reply:
Still doesn't throw?